six/modules/c++/cphd/source/Wideband.cpp

```diff
@@ -252,26 +252,26 @@ Wideband::Wideband(mem::SharedPtr<io::SeekableInputStream> inStream,
                    sys::Off_T startWB,
                    sys::Off_T sizeWB) :
     mInStream(inStream),
-    mData(data),
+    mData(&data),
     mWBOffset(startWB),
     mWBSize(sizeWB),
-    mElementSize(getNumBytesPerSample(mData.getSignalFormat())),
-    mOffsets(mData.getNumChannels())
+    mElementSize(mData->getNumBytesPerSample()),// THE ISSUE IS OVERLOADING THIS FUNCTION
```

Comment:
I don't understand this comment

Reply:
Was just a TODO for me. Hadn't removed the comments yet.